R/sfreemap.R

```diff
@@ -9,7 +9,7 @@ sfreemap <- function(tree, tip_states, Q=NULL, type="standard", model="SYM", met
     # Should this program run in parallel?
     if (hasArg(parallel)) {
         parallel <- list(...)$parallel
-        if (all(parallel, !support_parallel_mode())) {
+        if (any(parallel, !support_parallel_mode())) {
```

Comment:
This is actually correct I think, I want to show a warning if the user requested the parallel mode but it is not available in the operating system. So it should be the same case as before:
```
if (isTRUE(parallel) && support_parallel_mode()) {
```